EIPS/eip-1523.md

```diff
@@ -3,7 +3,7 @@ eip: 1523
 title: Standard for Insurance Policies as ERC-721 Non Fungible Tokens
```

Comment:
```suggestion
title: Standard for Insurance Policies as ERC-721 Non Fungible Tokens
description: A standard interface for insurance policies, based on ERC 721.
```
Also please remove the `## Simple Summary` section (it is replaced by description field).